src/utils/is.tsx

```diff
@@ -0,0 +1,16 @@
+export const isLink = (value: string) => value?.includes('http');
+
+export const isEmpty = (value: unknown) => {
+  if (!value && value !== false) return true;
+  if (Array.isArray(value) && value.length === 0) return true;
+  else return false;
+};
+
+export const isBoolean = (value: unknown) => typeof value === 'boolean';
+
+export const isValue = (value: any) => {
+  if (Array.isArray(value)) return 'array';
+  if (isBoolean(value)) return 'boolean';
+  if (isLink(value)) return 'url';
+  return 'string';
```

Comment:
What about an object? Is this a potential case?

Reply:
Not implemented yet. I will add this in new commit.